indico/modules/rb_new/operations/blockings.py

```diff
@@ -57,16 +58,41 @@ def get_room_blockings(timeframe=None, created_by=None, in_rooms_owned_by=None):
     return query.all()
 
 
+def filter_blocked_rooms(blocked_rooms, overridable_only=False, nonoverridable_only=False, explicit=False):
```

Comment:
if both only args are true, the result isn't what you'd expect (not that this combination would make sense..)

Reply:
Why not? It will return an empty list and there are no blockings that are both overridable and nonoverridable

Reply:
Indeed, nevermind.